packages/block-library/src/gallery/block.json

```diff
@@ -73,8 +29,25 @@
 		"sizeSlug": {
 			"type": "string",
 			"default": "large"
+		},
+		"allowAlign": {
+			"type": "boolean",
+			"default": false
+		},
```

Comment:
We can replace this with the new `layout` API introduced in #26380.

Reply:
Nice, thanks for the link, have refactored to use this.